Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM fundocker/edxapp:hawthorn.1-2.0.1
+
+ARG UID=1000
+ARG GID=1000
+
+# Switch back to a priviledged user to perform base installation
+USER root:root
+
+# Add the non-privileged user that will run the application
+RUN groupadd --gid ${GID} edx && \
+    useradd --uid ${UID} --gid ${GID} --home /edx edx
```

Comment:
Why do we need to do this and not in fun-platform?

If necessary, edx is not a good name for this, why not use edxapp?

Reply:
We are running this image only for local development purpose (and not production + development). This image is not supposed to be working in OpenShift or other platforms that assign a random non-privileged user ID to run a container. Hence, when working with this image, it's more **comfortable** to rely on an existing user (when you log in the container, etc.) than a random one activated _via_ an entrypoint (that is therefore not required here).

Moreover (even if it's not required in our case), the `COPY --chown` docker statement requires an existing system user (you cannot use the entrypoint trick during the build).

Concerning the group/username, `edx:edx` seemed relevant to me, but `edxapp` is also fine.

Reply:
> ok! I think this explanation needs to go somewhere in the code!

:thinking: Sure? 

> why not use edxapp?

Should I switch to it?

Reply:
edxapp for user: yes it's the common openedx practice.

Reply:
> :thinking:  Sure?
I meant in the repo... doc?

Reply:
I've added a header mentioning this in the `Dockerfile`.